internal/server/handlers.go

```diff
@@ -324,7 +325,13 @@ func createKubeconfig(clusterName, clusterControlPlaceAddress string) echo.Handl
 			return err
 		}
 
-		kubeCfg := kubeconfig.CreateKubeconfigYAMLForUser(c.Request().Context(), ac.Kubeclient, clusterName, clusterControlPlaceAddress, r.Username)
+		//required to make the API backward-compatible. Initially the API had only the username parameter
+		//this allows gradual migration of clients/scrips/e2e-tests that assume the request has only username parameter
```

Comment:
I think it should be removed the info regarding the PR, so after the merge, we have cohesive inline documentation.

Reply:
done 👍 
Feel free to give more suggestion/requirements if needed 💯